src/FileWriterTask.cxx

```diff
@@ -71,4 +81,25 @@ void FileWriterTask::file_flush() {
     impl->hdf_file.flush();
 }
 
+uint64_t FileWriterTask::id() const { return _id; }
+
+std::string const &FileWriterTask::hdf_filename() const {
```

Comment:
It is in [src/Master.cxx:89](https://github.com/ess-dmsc/kafka-to-nexus/blob/master_status/src/Master.cxx#L89)